hydromt/models/model.py

```diff
@@ -109,6 +108,8 @@ def __init__(
             {"region": {"type": "ModelRegionComponent"}}, components
         )
 
+        self.target_crs = CRS.from_user_input(target_model_crs)
```

Comment:
I would say that this will become model CRS, it's the target for the create methods as it can differ from the region crs but in the end it will be model crs. I would also allow to use the keyword "utm"  to let hydromt find the closest projected CRS using crs = gis_utils.parse_crs(crs, bbox=geom.total_bounds).
But you do need a total_bounds for that which you don't have at that stage. Let's leave it at this for now but we will need that when working on the final region PR with Marnix.